libs/ui/src/components/navigator/navigator.tsx

```diff
@@ -83,6 +84,7 @@ export const Navigator: FC = () => {
   useActiveTokenList();
   useTokensPriceInfo();
   useLoadSentCollectiblesBalance();
+  useResetLoading(showLoader);
```

Comment:
mb move `const showLoader = useShowLoaderSelector();` inside `useResetLoading`?  in this case `useResetLoading` will be totally independent